src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -147,7 +147,7 @@ public static int Main(string[] args)
                 catch (Exception ex)
                 {
                     //log.Error("Exception in RemoteTestAgent", ex);
-                    Console.WriteLine("Exception in RemoteTestAgent", ex);
+                    Console.WriteLine("Exception in RemoteTestAgent\r\n{0}", ex);
```

Comment:
Actually, if the other one is that way, then I guess this one is too. I don't like the format, but let's keep it consistent :smile: 

I will merge as-is, thanks.